ynr/apps/moderation_queue/tests/test_changes_for_review.py

```diff
@@ -374,6 +365,11 @@ def test_reverted_user_edit_needs_review(self, change_metadata=True):
             1,
         )
 
+        self.assertEqual(
+            LoggedAction.objects.needs_review().get().flagged_type,
+            "needs_review_due_to_reverted_edit_by_another_user",
```

Comment:
This comparison is a bit unnecessary as it is doing the same as the assert above but just in a different way. You can also make the `filter` more specific, by look for a logged action that is related to the `person` and `user` that you expect it to be

Reply:
Good idea, will do.